tests/budgeting/test_proposals_api_filtering.py

```diff
@@ -373,7 +373,9 @@ def test_proposal_ordering_filter(
                   kwargs={'module_pk': module.pk})
 
     # queryset is ordered by created
-    response = apiclient.get(url)
+    querystring = '?ordering=-created'
+    url_tmp = url + querystring
+    response = apiclient.get(url_tmp)
```

Comment:
before created unintentionally was default :see_no_evil:

Reply:
but that's still the case, or did I miss the fix?

Reply:
No, it should not (without the ?ordering=-created). The fix is the ordering property in the ProposalViewSet, which sets the default ordering if the ordering filter hasnt been applied.. https://www.django-rest-framework.org/api-guide/filtering/#specifying-a-default-ordering

Reply:
Right, you also order them here by created on purpose for the test... :+1: